logstash_async/constants.py

```diff
@@ -40,7 +40,7 @@ class Constants(object):
     # unless you know what you are doing
     FORMATTER_LOGSTASH_MESSAGE_FIELD_LIST = [
         '@timestamp', '@version', 'host', 'level', 'logsource', 'message',
-        'pid', 'program', 'type', 'tags']
+        'pid', 'program', 'type', 'tags', "@metadata"]
```

Comment:
Sorry for being picky, but would you mind using single quotes as for the other list elements?

Reply:
Updated commits to resolve this. Solutions like Black may be better instead of reviewing individual items like this in future